models/upload_sessions.go

```diff
@@ -68,6 +66,8 @@ const (
 	TreasureBuried
 )
 
+var StoragePeg = 64 // GB per year per PRL; TODO: query smart contract for real storage peg
```

Comment:
Or you could make it decimal number rather than int

Reply:
Okay, I'll change it.